content/en/data_jobs/airflow.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Enable Data Jobs Monitoring for Apache Airflow
+is_beta: true
+private: true
+aliases:
+    - /data_jobs/airflow/
+    - data_jobs/airflow/
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+{{< callout url="#" header="false" btn_hidden="true" >}}
+  Data Jobs Monitoring for Apache Airflow is in preview.
+{{< /callout >}}
+
+[Data Jobs Monitoring][1] gives visibility into the performance and reliability of workflows run by Apache Airflow DAGs.
+
+{{< tabs >}}
+{{% tab "Astronomer" %}}
+
+<div class="alert alert-warning">
+For Astronomer customers using Astro, the following setup may affect the lineage feature offered by Astro. see <a href=https://www.astronomer.io/docs/learn/airflow-openlineage#lineage-on-astro>this</a> for context.
+</div>
+
+## Requirements
+
+* [Apache Airflow 2.8.0][1] or later is required.
+* [apache-airflow-providers-openlineage][2] 1.12.0 or later is required.
+
+## Setup
+
+Data Jobs Monitoring is supported for Apache Airflow deployment with [apache-airflow-providers-openlineage][2] installed.
+
+To get started, follow the instructions below.
+
+1. Ariflow provider `openlineage` is already installed in [Astro Runtimes][3]. Customize the provider version by adding the following to your `requirements.txt` file into your [Astro project][4].
```

Comment:
Should we directly use the version installed by astronomer ? and not show how to customize it ?